### PR TITLE
Create a gpu project pool + bump janitor concurrency again

### PR DESCRIPTION
**Pull request by krzyzacy**
(Added a todo to get rid of the maximum janitor count in the future)

/assign @mindprince @BenTheElder

### CODE REVIEW

File: boskos/janitor/deployment.yaml [hunk 1]
```diff
@@ -15,10 +15,10 @@ spec:
       terminationGracePeriodSeconds: 300
       containers:
       - name: boskos-janitor
-        image: gcr.io/k8s-testimages/janitor:v20170830-c0d24857
+        image: gcr.io/k8s-testimages/janitor:v20170921-b5302345
         args:
         - --service-account=/etc/service-account/service-account.json
-        - --resource-type=gce-project,gke-project
+        - --resource-type=gce-project,gke-project,gci-qa-project,gpu-project
```
Comment (krzyzacy) [inline boskos/janitor/deployment.yaml]: (the addition with `gci-qa-project` is actually deployed but I guess I have never committed it :-( )

File: boskos/resources.json [hunk 2]
```diff
@@ -448,5 +448,15 @@
     ],
     "state": "dirty",
     "type": "gci-qa-project"
+  },
+  {
+    "names": [
```
Comment (mindprince) [inline boskos/resources.json]: Do you plan to move the existing projects to this pool in a separate PR?

Comment (krzyzacy) [reply boskos/resources.json]: We need to do things like whitelist gke, etc, I'll have another PR to create the 4 1.8-gpu jobs to use this pool to start with.

**Comment by BenTheElder** [comment]
whoo, let's get everything on boskos + prow :-)
/lgtm

**Pull request merged**